app/views/customers/index.html.erb

```diff
@@ -1,39 +1,30 @@
-<p id="notice"><%= notice %></p>
+<%= content_for :table_title do %>Customers<% end %>
 
-<h1>Customers</h1>
+<%= content_for :table_head do %>
+  <th>Full name</th>
+  <th>Phone</th>
+  <th>Email</th>
+<% end %>
 
-<table>
-  <thead>
+<%= content_for :table_body do %>
+  <% @customers.each do |customer| %>
```

Comment:
Just head's up, this blows up sometimes if the `@customers` variable isn't set (which it isn't always). It probably is better to wrap an `if` statement around it.